x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience_content.test.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { setMockValues } from '../../../../__mocks__/kea.mock';
+import { setMockSearchContextState } from './__mocks__/hooks.mock';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+import { EuiFlexGroup } from '@elastic/eui';
+// @ts-expect-error types are not available for this package yet
+import { Results } from '@elastic/react-search-ui';
+
+import { ResultView } from './views';
+import { SearchExperienceContent } from './search_experience_content';
+
+describe('SearchExperienceContent', () => {
+  const searchState = {
+    resultSearchTerm: 'searchTerm',
+    totalResults: 100,
+    wasSearched: true,
+  };
+  const values = {
+    engineName: 'engine1',
+    isMetaEngine: false,
+    myRole: { canManageEngineDocuments: true },
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(values);
+    setMockSearchContextState(searchState);
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<SearchExperienceContent />);
+    expect(wrapper.find(EuiFlexGroup).length).toBe(1);
+  });
+
+  it('passes engineName to the result view', () => {
+    const props = {
+      result: {
+        foo: {
+          raw: 'bar',
+        },
+      },
+    };
+
+    const wrapper = shallow(<SearchExperienceContent />);
+    const resultView: any = wrapper.find(Results).prop('resultView');
+    expect(resultView(props)).toEqual(<ResultView engineName="engine1" {...props} />);
+  });
+
+  it('renders pagination', () => {
+    const wrapper = shallow(<SearchExperienceContent />);
+    const pagination = wrapper.find('Pagination').at(0).dive();
+    expect(pagination.find(EuiFlexGroup).length).toBe(1);
```

Comment:
Same as above, I don't think we lose anything by simplifying this to

```suggestion
    const pagination = wrapper.find('Pagination').toHaveLength(1);
```

That's combined with my suggestion to pull `<Pagination />` out to its own component file, that way it also gets its own test/coverage

Reply:
af03b6e22b1aed01591e29e743828ef83431c62e